pages/settings.php

```diff
@@ -73,6 +74,13 @@
 $n['field'] = '<input type="checkbox" id="rex-developer-delete" name="config[delete]" value="1" ' . ($this->getConfig('delete') ? ' checked="checked"' : '') . ' />';
 $formElements[] = $n;
 
+if (rex_addon::exists('yform') && rex_addon::get('yform')->isInstalled()) {
```

Comment:
```suggestion
if (rex_addon::get('yform')->isAvailable()) {
```

Hier auch weiter nach oben verschieben nach "actions".